ios/FluentUI/Command Bar/CommandBarTokens.swift

```diff
@@ -16,67 +16,28 @@ public struct ButtonDynamicColors {
 
 /// Design token set for the `CommandBar` control.
 public class CommandBarTokens: ControlTokens {
-    /// Creates an instance of `CommandBarTokens`.
-    public override init() {
-        super.init()
-    }
+    open var backgroundColor: DynamicColor { aliasTokens.backgroundColors[.neutral1] }
 
-    /// Creates an instance of `CommandBarTokens` with optional token value overrides.
-    convenience public init(backgroundColor: DynamicColor? = nil,
-                            groupBorderRadius: CGFloat? = nil,
-                            groupInterspace: CGFloat? = nil,
-                            itemBackgroundColor: ButtonDynamicColors? = nil,
-                            itemFixedIconColor: DynamicColor? = nil,
-                            itemIconColor: ButtonDynamicColors? = nil,
-                            itemInterspace: CGFloat? = nil) {
+    open var groupBorderRadius: CGFloat { globalTokens.borderRadius[.xLarge] }
 
-        self.init()
+    open var groupInterspace: CGFloat { globalTokens.spacing[.medium] }
```

Comment:
What would be the story if the UIKit control like the CommandBar had a style or size enumeration?
Because it doesn't have a state object like the SwiftUI controls, are you proposing that the tokens object would have an unowned reference to the control?

Reply:
With the new approach, it would just push values. No more callbacks!